gtwrap/pybind_wrapper.py

```diff
@@ -319,76 +318,3 @@ def wrap(self):
             wrapped_namespace=wrapped_namespace,
             boost_class_export=boost_class_export,
         )
-
-
-def main():
```

Comment:
I think GTSAM will need to be adapted for this, right?

Reply:
I'll be checking for compatibility issues when I pull the lastest subtree. :)